### ISSUE TITLE
Stochastic Weighted Average (SWA) support in torch 1.6

### ISSUE THREAD

**Issue by glenn-jocher** [ultralytics/yolov5#793]
## 🚀 Feature
SWA for YOLOv5. This would be an experiment to determine if SWA offers improvements over the current EMA implementation.
https://pytorch.org/blog/pytorch-1.6-now-includes-stochastic-weight-averaging/

Tests would be YOLOv5l COCO training from scratch to 300 epochs with 1) no averaging, 2) EMA, 3) SWA.

It should be noted that the current cosine LR scheduler provides similar functionality to EMA and SWA, but in the last YOLOv3 tests we ran when implementing EMA we still saw improvements over the baseline 'no averaging' scenario even with the cosine LR scheduler in place. See https://github.com/rwightman/pytorch-image-models/issues/102#issuecomment-601424476
